drift/historical_sys_profile_service_interface.py

```diff
@@ -0,0 +1,27 @@
+from urllib.parse import urljoin
+
+from drift import metrics
+from kerlescan import config
+from kerlescan.service_interface import fetch_data
+from kerlescan.constants import AUTH_HEADER_NAME, PIT_SVC_ENDPOINT
```

Comment:
For consistency, should PIT_SVC_ENDPOINT be renamed?

Reply:
it does. I'll rename it in a different PR so I can update kerlescan at the same time. Trying to not break too many things at once :)